extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Anchor.fm integration.
+ *
+ * @since 9.0.0
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\AnchorFm;
+
+use Automattic\Jetpack\Blocks;
+use Jetpack_Podcast_Helper;
+
+const FEATURE_NAME = 'anchor-fm';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers Anchor.fm integration for the block editor.
+ */
+function register_block() {
+	Blocks::jetpack_register_block(
+		BLOCK_NAME,
+		array(
+			'plan_check' => false,
+		)
+	);
```

Comment:
I think you could skip this altogether since it's `false` by default.

```suggestion
	Blocks::jetpack_register_block( BLOCK_NAME );
```